python/packages/kagent-adk/src/kagent/adk/_agent_executor.py

```diff
@@ -175,14 +189,6 @@ async def _handle_request(
 
         await runner.session_service.append_event(session, system_event)
 
-        current_span = trace.get_current_span()
```

Comment:
Shouldn't we keep this? I am wondering if we can miss attributes with the processor-only approach for certain spans?

Reply:
I have modified this to now just use a parent span. The span (see L169), has an instrumentation scope name of "kagent-adk" and a span name of "kagent_a2a_request". This span shows up in my testing, with the custom attributes as the parent span for all subsequent spans related to this request. Unlike the span processor though, this will propagate these attributes to any child spans created. This way is very simple.

Reply:
I also updated the span name to be execute and move the span creation to a place that can be consistent between kagent-adk, kagent-langgraph, and kagent-crewai.